src/test/java/org/jabref/logic/importer/util/GrobidServiceTest.java

```diff
@@ -49,10 +49,14 @@ public void processValidCitationTest() throws IOException {
         assertTrue(responseRows[1].contains("author") && responseRows[1].contains("Derwing, Tracey and Rossiter, Marian and Munro, Murray"));
```

Comment:
I don't like this assertTrue stuff here.
Isn't there any way to have an assertEquals?
 I would propose a List.of(first line text, secondlinetext) and then compare it to  the response

Reply:
True, but not exactly related to this PR, so I opened #8003.